src/modules/registrypreview/RegistryPreviewExt/dllmain.cpp

```diff
@@ -274,7 +274,7 @@ class RegistryPreviewModule : public PowertoyModuleIface
             Trace::EnableRegistryPreview(false);
             Logger::trace(L"Disabling Registry Preview...");
 
-            // Yeet the Registry setting so preview doesn't work anymore
+            // Remove the Registry setting so preview doesn't work anymore
```

Comment:
I looked into the history of this. I think it originally wanted to be `Delete`, but then the code changed significantly to not actually use deletes, so `Remove` seems better

Reply:
FWIW, the various teams hanging around here and in Terminal consider `yeet` to be perfectly cromulent vocabulary. :grin:

Reply:
Can one of them _define_ it?

Reply:
Ha! We use it for "throw in" or "throw out" or "delete" or "remove" or, more broadly, "add to the project at the last second". It suggests some amount of indelicate _throwing_, like from this comic:

![image](https://github.com/user-attachments/assets/6656c518-487d-4eac-a23e-84a87d891a00)